README.md

```diff
@@ -23,21 +23,73 @@
 
 ## Introduction
 
-Jujutsu is a
-[Git-compatible](https://martinvonz.github.io/jj/latest/git-compatibility)
-[DVCS](https://en.wikipedia.org/wiki/Distributed_version_control). It combines
-features from Git (data model,
-[speed](https://github.com/martinvonz/jj/discussions/49)), Mercurial (anonymous
-branching, simple CLI [free from "the
-index"](https://martinvonz.github.io/jj/latest/git-comparison#the-index),
-[revsets](https://martinvonz.github.io/jj/latest/revsets), powerful
-history-rewriting), and Pijul/Darcs ([first-class
-conflicts](https://martinvonz.github.io/jj/latest/conflicts)), with features not
-found in most of them
-([working-copy-as-a-commit](https://martinvonz.github.io/jj/latest/working-copy),
-[undo functionality](https://martinvonz.github.io/jj/latest/operation-log),
-automatic rebase, [safe replication via `rsync`, Dropbox, or distributed file
-system](https://martinvonz.github.io/jj/latest/technical/concurrency)).
+Jujutsu is a powerful [version control system](https://en.wikipedia.org/wiki/Version_control)
+system for software projects. You use it to get a copy of the code, track
+changes to the code, and finally publish those changes for others to see and
+use. Jujutsu is unlike most other systems, because internally it abstracts
```

Comment:
I think "Jujutsu is unlike most other systems" should be the beginning of a new paragraph.

I also feel a bit iffy about the content of what would become this new paragraph, but it's not bad, and I haven't come up with any potential improvements.